src/index.html

```diff
@@ -5,9 +5,194 @@
+          <div class="header__action">
+            <a href="#" class="header__link">Call to order</a>
+          </div>
+        </div>
+
+        <nav class="nav header__nav">
+          <input type="checkbox" class="header__toggler" id="toggler">
+            <label for="toggler" class="header__toggler-label">
+              <span></span>
+            </label>
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="" class="nav__link nav__link_active">Home</a>
+            </li>
+            <li class="nav__item">
+              <a href="" class="nav__link"> About Us</a>
+            </li>
+            <li class="nav__item">
+              <a href="" class="nav__link">Gallery</a>
+            </li>
+            <li class="nav__item">
+              <a href="" class="nav__link">Consultation</a>
+            </li>
+            <li class="nav__item">
+              <a href="" class="nav__link">Testimonials</a>
+            </li>
+            <li class="nav__item">
+              <a href="" class="nav__link">Contacts</a>
+            </li>
+            <li class="nav__item nav__item_phone">
+              <a href="tel:+1-234-555-55-55" class="nav__link">+1 234 555-55-55</a>
+            </li>
+            <li class="nav__item nav__item_call">
+              <a class="nav__link nav__link_phone" href="#">Call to order</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main>
+      <div class="main-wrapper">
+        <section class="section-info">
+          <div class="info">
+            <h2 class="info__heading">Miami best<br> Real estate <a href="#" class="info__heading__link">here</a></h2>
+            <div class="info__text">
+              <h3 class="info__subheading">Over 50 years experience in love.</h3>
+              <p>Mazel and VA Tsukkerman in his <q>Analysis of musical works.</q>
+                Pointillism, which originated in the music of the early twentieth century,
+                microforms, found a distant historical.</p>
+              <p>The market leader in real estate since 2003. Awards Real Estate Company,
+                2010, 2011, 2013. More than 200 satisfied customers premium segment.
+                We focus only on new buildings and high-level doskanalno know this area
+                better than other specialists. Working directly with developers - so have
+                more useful information about the profitable deals.</p>
+              <p><strong>We will never be late!</strong></p>
+              <p><strong>Guaranteed secure transaction - 100%</strong></p>
+            </div>
+          </div>
+        </section>
+
+        <section class="section-gallery">
+          <div class="gallery">
+            <h2 class="gallery__heading">Gallery</h2>
+            <div class="photos photos__wide_left">
+              <img src="./images/photos/1_optimized.jpg" alt="Marina Palms, North Miami Beach" class="photos__img photos__img_1">
```

Comment:
переноси аттрибуты длинных тегов (исправь везде)
```suggestion
              <img
                src="./images/photos/1_optimized.jpg" 
                alt="Marina Palms, North Miami Beach" 
                class="photos__img photos__img_1"
              >
```